cms/bundles/tests/test_viewsets.py

```diff
@@ -320,6 +342,45 @@ def test_bundle_edit_view__manual_publish__happy_path(self):
         self.bundle.save(update_fields=["status"])
         self.post_with_action_and_test("action-publish", BundleStatus.PUBLISHED, self.bundle_index_url)
 
+    release_calendar_page_cases: ClassVar[list[tuple[str, ReleaseStatus, str]]] = [
```

Comment:
Why is this just sitting in the middle of nowhere? I think I gave feedback on these kinds of things many times.